### PR TITLE
Johnsall/cycle6 readme

### PR DESCRIPTION
**Pull request by johnsall**
[XXXX-<Title> - Please use the Work Item number and Title as PR Name, not subtasks]

#### 📲 What

Updated the Readme to reflect changes from cycle 6

#### 🤔 Why

Why it's needed, background context.

#### 🛠 How

More in-depth discussion of the change or implementation.

#### 👀 Evidence

Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR

#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,160 @@
 The full documentation on Amido Stacks can be found [here](https://amido.github.io/stacks/).
 
-Java Spring boot Rest API with CQRS documentation can be found [here](https://amido.github.io/stacks/docs/workloads/azure/backend/java_cqrs/intro_java_cqrs).
\ No newline at end of file
+## Using the repository to build the Java Spring Boot Rest API application
+
+### Versioning
+
+This is version `1.0.0` of the `java-stacks-cqrs-events` project.
```
Comment (radoslaw-orlowski) [inline README.md:7]: Wouldn't this be a pain to maintain the correct version?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,160 @@
...
+
+### Customised solution
+
+To customise the namespaces and create an application for your company please visit the full [documentation](https://stacks.amido.com/docs/)
```
Comment (radoslaw-orlowski) [inline README.md:11]:
```suggestion
To customise the namespaces and create an application for your company, please visit the full [documentation](https://stacks.amido.com/docs/).
```

**Review by radoslaw-orlowski — APPROVED** [review]
One question, otherwise all good.

File: README.md (hunk 1 above)
Comment (johnsall) [reply README.md:7]: You're right, I'll remove it or basically call it v2

**Pull request merged**